Userland/Applications/PixelPaint/FilterTreeModel.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, Tobias Christiansen <tobyase@serenityos.org>
+ * Copyright (c) 2022, Linus Groh <linusg@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Filters/Filter.h"
+#include "ImageEditor.h"
+#include <AK/NonnullOwnPtr.h>
+#include <LibGUI/TreeViewModel.h>
+
+namespace PixelPaint {
+
+class FilterNode final : public GUI::TreeViewModel::Node {
```

Comment:
Any reason why the file is called FilterTreeModel while the only class in the file is named FilterNode?

Reply:
Because the tree we create is called `m_filter_tree`, and this file contains the infrastructure (`FilterNode` + `create_filter_tree_model()`) to create the model (`GUI::TreeViewModel`) for said tree :^)
I don't think files always need to match class names.